javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentTracerProviderConfigurer.java

```diff
@@ -44,6 +46,23 @@ public void configure(SdkTracerProviderBuilder sdkTracerProviderBuilder) {
     sdkTracerProviderBuilder.addSpanProcessor(new AddThreadDetailsSpanProcessor());
 
     maybeConfigureExporterJar(sdkTracerProviderBuilder);
+    maybeEnableLoggingExporter(sdkTracerProviderBuilder);
+  }
+
+  private static void maybeEnableLoggingExporter(SdkTracerProviderBuilder builder) {
+    if (Config.get().isAgentDebugEnabled()) {
+      // don't install another instance if the user has already explicitly requested it.
+      if (loggingExporterIsAlreadyConfigured()) {
+        builder.addSpanProcessor(SimpleSpanProcessor.create(new LoggingSpanExporter()));
+      }
+    }
+  }
+
+  private static boolean loggingExporterIsAlreadyConfigured() {
+    // todo: is this really a reliable way to detect this?
+    Config config = Config.get();
+    return !config.getProperty("otel.traces.exporter", "").equalsIgnoreCase("logging")
+        && !config.getProperty("otel.exporter", "").equalsIgnoreCase("logging");
```

Comment:
I don't think we support this property anywhere anymore.

Reply:
removed